### PR TITLE
Added begin and end offset param

### PR DESCRIPTION
**Pull request by yazebochan**

### CODE REVIEW

File: kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml [hunk 1]
```diff
@@ -290,6 +290,28 @@ paths:
                 items:
                   $ref: '#/components/schemas/ConsumerGroup'
 
+  /api/clusters/offsets/{clusterName}:
```
Comment (germanosin) [inline kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:293]: Why do we need special endpoint? Could we add this to exist topic partition list?

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java [hunk 2]
```diff
@@ -351,4 +351,40 @@ private InternalClusterMetrics collectTopicsMetrics(Map<String,InternalTopic> to
                 })
             );
     }
+
+    public Mono<Map<String, BigDecimal>> getOffsets (AdminClient ac, KafkaCluster c) {
```
Comment (germanosin) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java:355]: What is String? TopicPartition?

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java [hunk 3]
```diff
@@ -351,4 +351,40 @@ private InternalClusterMetrics collectTopicsMetrics(Map<String,InternalTopic> to
...
+                                                    .collect(Collectors.toList()))
+                                            .collect(Collectors.toList()))
+                            .collect(Collectors.toList())
+                            .stream().flatMap(List::stream).collect(Collectors.toList())
+                            .stream().flatMap(List::stream).collect(Collectors.toList());
+                    return getTopicPartitionOffset(c, tps);
+                });
+    }
+
+    private Mono<List<Map<String, List<Integer>>>> getTopicPartitionList(AdminClient ac) {
```
Comment (germanosin) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java:372]: I don't think we need all these offsets in one time

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java (hunk 2 above)
Comment (yazebochan) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java:355]: Topic-Partition-max/min to avoid creating new object in swagger

**Pull request merged**